tests/test_main.py

```diff
@@ -505,4 +505,4 @@ def test_movie_session_service_delete_movie_session_by_id(database_data):
     ) == [
         (datetime.date(2017, 8, 19), "Cheap", "Titanic"),
         (datetime.date(2021, 4, 3), "VIP", "The Good, the Bad and the Ugly"),
-    ]
+    ]
```

Comment:
please revert changes here

Reply:
These are default tests, but for some reason they are displayed as modified

Reply:
you must have added a blank line or something